server.py

```
@@ -0,0 +1,114 @@
+from flask import Flask,jsonify, request
+import os
+import shutil
+import pathlib
+
+
+app = Flask(__name__)
+
+#==============================================heartbeat<3=========================
+
+@app.route('/heartbeat' , methods=['GET', 'POST'])
+def heartbeat():
+    return "204"
+
+#==============================================reading file=========================
+@app.route('/GET/<string:pathToFile>', methods=['GET'])
+def GetFile(pathToFile):
+    if os.path.exists(pathToFile):
+        f = open(pathToFile, "r")
+        text=f.read()
+        f.close()
+        return text
+    else:
+        return "404"
```

When you want to return an HTTP error code, use the abort function:
```python
from flask import abort
abort(404)
```